rust/core/src/driver_manager.rs

```diff
@@ -1894,6 +1894,7 @@ mod tests {
     #[cfg_attr(not(feature = "driver_manager_test_lib"), ignore)]
     fn test_load_driver_env() {
         // ensure that we fail without the env var set
+        env::remove_var("ADBC_CONFIG_PATH");
```

Comment:
With tests running parallel (default) and other tests also setting this env var there is a race. Instead we could use https://crates.io/crates/temp-env or https://crates.io/crates/serial_test.

Reply:
Thanks!
temp-env looks ideal.

Reply:
I opened #3152